public/video-ui/src/components/ManagedForm/ManagedField.js

```diff
@@ -100,6 +100,10 @@ export class ManagedField extends React.Component {
         _set(this.props.fieldLocation, null, this.props.data)
       );
     }
+
+    if (this.props.updateSideEffects) {
```

Comment:
This function is used in the [tagpicker](https://github.com/guardian/media-atom-maker/blob/0d2a975d6ffde2ab6fd8a83a3978361271c02d0c/public/video-ui/src/components/FormFields/TagPicker.js#L346) as well but inside the component to copy tags. Does the function call here cause errors from the tagpicker, especially since the function in the tagpicker does not take any arguments.

Reply:
good call, will check!

Reply:
tag picker is working as usual :)